entry/src/block_component.rs

```diff
@@ -902,39 +976,43 @@ impl BlockFooterV1 {
 
     /// Deserializes from bytes with validation.
     fn from_bytes(data: &[u8]) -> Result<Self, BlockComponentError> {
-        const TIMESTAMP_SIZE: usize = 8;
-        const LENGTH_SIZE: usize = 1;
-        const HEADER_SIZE: usize = TIMESTAMP_SIZE + LENGTH_SIZE;
-
-        if data.len() < HEADER_SIZE {
+        if data.len() < Self::HEADER_SIZE {
             return Err(BlockComponentError::InsufficientData);
         }
 
         // Read timestamp
-        let time_bytes: [u8; TIMESTAMP_SIZE] = data[..TIMESTAMP_SIZE]
+        let time_bytes: [u8; Self::TIMESTAMP_SIZE] = data[..Self::TIMESTAMP_SIZE]
```

Comment:
```suggestion
        let time_bytes = data[..Self::TIMESTAMP_SIZE]
```

inference works here.